src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const cssObject = {};
+
+  const noSpacesArray = sourceString.split(';');
+  const arrayOfArrays = noSpacesArray
+    .map((element) => element.split(':'))
+    .filter((element) => element.length === 2)
+    .map((element) => [element[0].trim(), element[1].trim()]);
+
+  arrayOfArrays.forEach((element) => {
+    cssObject[element[0]] = element[1];
```

Comment:
The forEach loop is correctly used here to modify the existing 'cssObject'. However, you can further optimize the code by incorporating this logic into the map operation, thus eliminating the need for an explicit loop.